backup.html

```diff
@@ -54,75 +54,96 @@ <h2>Backup center</h2>
             &nbsp;&nbsp;&nbsp;
             <button id="backup-clear-button" class="btn btn-default"><i class="fa fa-eraser"></i> Clear</button>
             <br><br>
-           
+
             <div class="placeholder">BIP32 extended master private key</div>
-            <textarea id="backup-xpriv" class="text-input" rows="3" cols="70" wrap="hard" spellcheck="false" readonly></textarea>
+            <output id="backup-xpriv" class="extended-keys"></output>
```

Comment:
I thought it was more fitting than a text area, since it contains the output of the key generation.

Reply:
I remember that another reason for it was space, since we will add additional coins in the future. I discussed this briefly with Marko, though he did not recommend this particular field.